### DISCUSSION TITLE
Default: When SHLL starts WEB, open IP access

### DISCUSSION THREAD

**Discussion by pgy866** [OrchardCMS/OrchardCore#10394] [Ideas]
//改变配置文件
//Change configuration file

https://github.com/OrchardCMS/OrchardCore/blob/main/src/OrchardCore.Cms.Web/appsettings.json
https://github.com/pgy866/OrchardCore/blob/8ff92f90ab4d6ed348c7110ccb7faecf4054d3f3/src/OrchardCore.Cms.Web/appsettings.json

```
     //默认：SHLL启动WEB时，开启IP访问 
    // Default: When SHLL starts WEB, open IP access
 "AllowedHosts": "*",
  "Kestrel": {
    "EndPoints": {
      "Http": {
       "Url": "http://*:5000"
       },
      "Https": {
       "Url": "https://*:5001"
      }
    }
  },
```

**Comment by Skrypt** [comment]
You should definitely not use "AllowedHosts": "*" as it allows email spoofing from any domains. At least when using Kestrel. IIS should not be an issue.

See : https://github.com/OrchardCMS/OrchardCore/pull/10388/files

As for the Kestrel endpoints I don't think it is necessary to be by default in the appsettings file.

**Reply by pgy866** [reply]
Sorry, I ignored the security, mainly because shll can only be accessed by default at http://localhost:5000 after it is started

抱歉，我把安全忽略啦，主要是shll启动后默认只可以http://localhost:5000，访问它

@Skrypt

**Reply by Skrypt** [reply]
Then you can set `"AllowedHosts": "yourwebsitedomain.something;localhost"` and it should work.

**Reply by hishamco** [reply]
Similar to what Seb suggested here #10388
